tests/factories/user_category_factories.py

```diff
@@ -0,0 +1,12 @@
+from tests.factories import *
+
+from sourcemash.models import UserCategory, Category
+from user_factories import UserFactory
+
+class UserCategoryFactory(SQLAlchemyModelFactory):
+    FACTORY_FOR = UserCategory
+    FACTORY_SESSION = db.session
+
+    user = factory.SubFactory(UserFactory)
+    category = Category("category")
```

Comment:
We don't have a category factory and there isn't really a use for one (since we have itemWithCategory), so it just makes a Category here for this.